editor.planx.uk/src/pages/FlowCard.tsx

```diff
@@ -160,6 +161,24 @@ const FlowCard: React.FC<FlowCardProps> = ({
       });
   };
 
+  const isSubmissionService = flow.publishedFlows?.[0]?.hasSendComponent;
+
+  const statusVariant =
+    flow.status === "online" ? StatusVariant.Online : StatusVariant.Offline;
+
+  const displayTags = [
+    {
+      type: FlowTagType.Status,
+      displayName: statusVariant,
+      shouldAddTag: statusVariant,
```

Comment:
This is just working because this string is always truthy - it would be simpler / clearer to make this `true` to indicate it's always displayed.